wormchain/app/app.go

```diff
@@ -135,6 +140,15 @@ func GetWasmOpts(app *App, appOpts servertypes.AppOptions) []wasm.Option {
 	// add the custom wormhole query handler
 	wasmOpts = append(wasmOpts, wasmkeeper.WithQueryPlugins(wormholemodulekeeper.NewCustomQueryHandler(app.WormholeKeeper)))
 
+	// Move custom query of token factory to stargate, still use custom msg which is tfOpts[1]
+	var bankBaseKeeper bankkeeper.BaseKeeper
+	bankBaseKeeper, ok := app.BankKeeper.(bankkeeper.BaseKeeper)
```

Comment:
nit: don't need the `var` declaration on line 144, can declare directly on line 145.

Reply:
Will remove